### PR TITLE
OSDOCS-3189: CMP: Fix Steps 1-3 in Installing the CO using the CLI

### PR DESCRIPTION
**Pull request by kmccarron-rh**
https://issues.redhat.com/browse/OSDOCS-3189
Affects 4.6+

Best way to view this change is the direct preview [here](https://deploy-preview-40778--osdocs.netlify.app/openshift-enterprise/latest/security/compliance_operator/compliance-operator-installation.html#installing-compliance-operator-cli_compliance-operator-installation).

### CODE REVIEW

**Comment by rhmdnd** [comment]
Looks good and reads much nicer. Thanks!

**Comment by kmccarron-rh** [comment]
@pdhamdhe ptal, thanks!

File: modules/compliance-operator-cli-installation.adoc [hunk 1]
```diff
@@ -28,15 +23,16 @@ metadata:
...
   name: openshift-compliance
 ----
-
-. Create the `OperatorGroup` object YAML file by running:
+. Create the `Namespace` object:
 +
 [source,terminal]
 ----
-$ oc create -f <file-name>.yaml
+$ oc create -f namespace-object.yaml
```
Comment (mburke5678) [inline modules/compliance-operator-cli-installation.adoc:30]: I like the `<file-name>.yaml` structure as it indicates the user can use any name. But I think an admin user can follow this procedure.

**Comment by mburke5678** [comment]
@kmccarron-rh One minor nit. Feel free to disregard. Otherwise LGTM

File: modules/compliance-operator-cli-installation.adoc (hunk 1 above)
Comment (kmccarron-rh) [reply modules/compliance-operator-cli-installation.adoc:30]: Ok, ty!- I will leave it this way for now - it was patterned after this [example](https://docs.openshift.com/container-platform/4.9/authentication/ldap-syncing.html#ldap-auto-syncing_ldap-syncing-groups).

**Pull request merged**